app/views/users/index.html.erb

```diff
@@ -7,7 +7,7 @@
     </div>
     <div class="lg-12-3">
       <% unless current_user %>
-        <%= link_to user_omniauth_authorize_path(:github), class: "btn btn-github" do %>
```

Comment:
@enderahmetyurt yalnız eski şekliyle de localhost'ta çalışmıyor. github api localhost'ta çalışmadığı için olsa gerek. stage'de denedin mi hocam?